buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -107,6 +108,7 @@ object Libraries {
     const val composeConstraintLayout   = "androidx.constraintlayout:constraintlayout-compose:${Versions.composeConstraint}"
     const val composeRuntimeLiveData    = "androidx.compose.runtime:runtime-livedata:${Versions.compose}"
     const val dataStore                 = "androidx.datastore:datastore-preferences:${Versions.dataStore}"
+    const val exifInterface                      = "androidx.exifinterface:exifinterface:${Versions.exif}"
```

Comment:
despite sounding a bit picky/pixel perfect nerdie, can we level it up to the rest of dependencies to make it more obvious at first sight from which library comes? 🤓